README.md

```diff
@@ -45,20 +48,21 @@ instructions under "Running Old Versions" to display the old guides.
 After cloning this repo, install dependencies and run the demo app:
 
 ```bash
-$ npm install
-$ ember serve
+npm install
+bower install
```

Comment:
Let me check tomorrow, but my impression is that for the demo app, both are needed: npm installs some of the dependencies and bower takes care of the rest.